pkg/plugin/v3/scaffolds/internal/templates/config/api/webhook.go

```diff
@@ -82,8 +82,10 @@ package {{ .Resource.Version }}
 import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
-	{{- if or .Validating .Defaulting }}
+	{{- if .Validating }}
 	"k8s.io/apimachinery/pkg/runtime"
+	{{- end }}
+	{{- if or .Validating .Defaulting }}
```

Comment:
Hi @prafull01, 

Is this error faced with v2 as well? I understand that yes, so I think since it is bug fix should be made to v2+ too. 
Also, could not we check this scenario in the generate testdata as well to avoid this kind of issue in the future?

Reply:
I will fix this in v2 as well. I can add defaulting and programmatic-validation test scenario in generate testdata to avoid such issues in future.

Reply:
Done..